src/GraphQL/Queries/EntriesQuery.php

```diff
@@ -6,15 +6,15 @@
 use Statamic\Facades\Entry;
 use Statamic\Facades\GraphQL;
 use Statamic\GraphQL\Middleware\ResolvePage;
+use Statamic\GraphQL\Queries\Concerns\FiltersQuery;
 use Statamic\GraphQL\Types\EntryInterface;
 use Statamic\GraphQL\Types\JsonArgument;
-use Statamic\Support\Arr;
 use Statamic\Support\Str;
 use Statamic\Tags\Concerns\QueriesConditions;
 
 class EntriesQuery extends Query
 {
-    use QueriesConditions;
+    use QueriesConditions, FiltersQuery;
```

Comment:
Just `use FiltersQuery`. Remove `QueriesConditions`.

We'll `use` that from the trait instead.